src/views/bootablevolumes/list/BootableVolumesList.tsx

```
@@ -35,12 +38,13 @@ import '@kubevirt-utils/styles/list-managment-group.scss';
 const BootableVolumesList: FC = () => {
   const { ns: namespace } = useParams<{ cluster: string; ns: string }>();
   const { t } = useKubevirtTranslation();
-
+  const filteredClusters = useSelectedRowFilterClusters()
```

ESLint

[error] Insert `;`

(prettier/prettier)